roles/config-quay-enterprise/tasks/main.yml

```diff
@@ -128,6 +128,13 @@
 - name: Include firewall tasks
   include_tasks: firewall.yml
 
+- name: Stop quay to complete setup
```

Comment:
The stopping of quay and completing the setup process should most likely me moved into a `block` since we would only want to perform those actions if we in fact need to setup quay and not on subsequent runs